bears/python/YapfBear.py

```diff
@@ -0,0 +1,137 @@
+import sys
+
+from coalib.bears.LocalBear import LocalBear
+from yapf.yapflib.yapf_api import FormatFile
+from coalib.results.Result import Result
+from coalib.bearlib.spacing.SpacingHelper import SpacingHelper
+from coalib.results.Diff import Diff
+from coalib.misc.ContextManagers import prepare_file
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+class YapfBear(LocalBear):
+    """
+    Check and correct formatting of Python code using ``yapf`` utility.
+
+    See <https://github.com/google/yapf> for more information.
+    """
+    LANGUAGES = {"Python"}
```

Comment:
Does this support Python2 ? Since we have Python,Python3, Python2 I am wondering whether we should have all three listed or no? Someone else might be able to better answer this.